.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,6 +20,7 @@ If there is no corresponding issue number, fill in N/A where it says [fill_in_nu
 
 <!--
 Add videos/screenshots of the user-facing interface to demonstrate that the changes made in this PR work correctly.
+If proof cannot be provided include a sentence saying "No proof of changes needed because {{Reason}}" and the reason why proof of changes cannot be provided.
```

Comment:
@gp201 Instead of allowing contributors to decide whether PRs need to provide proof or not, should we explicitly write down the situation/cases where the contributor doesn't have to provide the proof?

I found the following cases:
1. Changes in CODEOWNER file to transfer code owner [We are going to have a script to automate this process.]
2. Automated pre-release PRs to update CHANGELOG, CONTRIBUTOR and AUTHORS files.

Do you know any other situation(s)?

(not sure, but can we assume only the autogenerated PR doesn't need Proof?)

Reply:
@DubeySandeep There are very few cases that do not have any "proof of changes". Most of those cases are autogenerated PR's. So I guess we can make the sentence like this.


```diff
- If proof cannot be provided include a sentence saying "No proof of changes needed because {{Reason}}" and the reason why proof of changes cannot be provided.
+ If proof cannot be provided include a sentence saying "No proof of changes needed because {{Reason}}" and the reason why proof of changes cannot be provided. ex: Autogenerated PR's
```

Reply:
If we are pretty sure that we only want to avoid this section for autogenerated PRs then should we write it:

`If the changes in your PRs are autogenerated via a script and you cannot provide proof for the changes the please leave a comment "No proof of changes needed because {{Reason}}"` or something around this?

Reply:
@DubeySandeep 

```
If the changes in your PRs are autogenerated via a script or you cannot provide proof for the changes then please leave a comment "No proof of changes needed because {{Reason}}".
```
and -> or?

Reply:
No, let's make it more specific. use `and`

Reply:
DONE